models.py

```diff
@@ -0,0 +1,181 @@
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.neural_network import MLPClassifier
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import optim
+from torch.autograd import Variable
+
+from joblib import dump, load
+from datetime import datetime
+import os, time
+
+from utils.wrappers import RobustRewardEnv
+
+import numpy as np
+import tensorflow as tf
+
+PARAMS = {
+        'max_steps':        20000,
+        'learning_rate':    1e-3,
+        'batch_size':       512,
+        'weight_decay':     1e-2,
+        'tensorboard_freq': 10,
+        'save_freq':        10,
+        }
+
+class Logger(object):
+    def __init__(self, log_dir):
+        self.writer = tf.summary.FileWriter(log_dir)
+    
+    def scalar_summary(self, tag, value, step):
+        summary = tf.Summary(value=[tf.Summary.Value(tag=tag, simple_value=value)])
+        self.writer.add_summary(summary, step)
+
+class ClassificationModel(object):
+	def __init__(self, dataset_name, env_name, q, seed=0, path='', hidden_size=20):
+		self.dataset_name = dataset_name
+		self.env_name = env_name
+		self.seed = seed
+		self.q = q
+		self.model_list = [MLPClassifier(hidden_layer_sizes=(hidden_size, hidden_size),
+										 random_state=self.seed, 
+										 max_iter=1000, 
```

Comment:
Note this doesn't use `PARAMS['max_steps']`

Reply:
The `PARAMS['max_steps']` were intended for Video Pinball training (cf. line 141). I agree that I could also use it for Hopper-v2, but they don't really have the same meaning in the two optimization processes.

Reply:
It would be preferable to name them differently, but this is fine for now.